toolchain/lc-compile/src/report.c

```diff
@@ -172,6 +205,7 @@ static void _Warning(long p_position, const char *p_message)
     {
         _PrintPosition(p_position);
         fprintf(stderr, "warning: %s\n", p_message);
+        _PrintContext(p_position);
```

Comment:
@peter-b I'm wondering why `_Warning`, `_ErrorS` and `_WarningS` all include the context but `_Error` doesn't.